src/ATen/common_with_cwrap.py

```diff
@@ -93,13 +93,21 @@ def signature(option, kwarg_only_count):
 
 def enumerate_options_due_to_default(declaration,
                                      allow_kwarg=True, type_to_signature=[], remove_self=True):
+
+    # Checks to see if an argument with a default keyword is a Tensor that
+    # by default can be NULL. In this case, instead of generating another
+    # option that excludes this argument, we will instead generate a single
+    # function call that allows for the Tensor to be NULL
```

Comment:
Can we also generate the version with fewer arguments? That is, one version has no argument. The other one takes an argument that can be null. This makes  it consistent with other default tags. I think just deleting the "and not is_nullabel_tensor_arg call" will do this.